tests/transformer/test_yield.py

```diff
@@ -12,14 +13,31 @@ def no_yield():
 
 @pytest.mark.parametrize(*c_exec)
 def test_yield(c_exec):
-    """It prevents using the `yield` statement."""
+    """`yield` statement should be allowed."""
     result = c_exec(YIELD_EXAMPLE)
-    assert result.errors == ("Line 2: Yield statements are not allowed.",)
-    assert result.code is None
+    assert result.errors == ()
+    assert result.code is not None
```

Comment:
It would be nice if the code actually gets executed like in https://github.com/zopefoundation/RestrictedPython/blob/master/tests/transformer/test_try.py#L9-L29

Reply:
should be fixed